frontend/js/util.js

```diff
@@ -248,6 +248,18 @@ var util = {
         return _.extend(object, _.object(_.map(properties, function (property) {
             return [property, definition(property)];
         })));
+    },
+
+    /**
+     * Capitalize a string
+     * @param {*} s The string to capitalize
```

Comment:
With my suggestion below, this should be

```suggestion
     * @param {string} s The string to capitalize
```